pandas/tests/indexes/interval/test_base.py

```diff
@@ -0,0 +1,82 @@
+import numpy as np
+import pytest
+
+from pandas import IntervalIndex, Series, date_range
+from pandas.tests.indexes.common import Base
+import pandas.util.testing as tm
+
+
+class TestBase(Base):
+    """
+    Tests specific to the shared common index tests; unrelated tests should be placed
+    in test_interval.py or the specific test file (e.g. test_astype.py)
+    """
+
+    _holder = IntervalIndex
+
+    def setup_method(self, method):
```

Comment:
can you use fixtures here instead? (can be a followup)

Reply:
Yeah, can do it as a followup.  Some of the machinery here is required for the `Base` class to work so will need to be changed across the board for all index tests inheriting from `Base`.